src/main/java/igrad/model/course/CourseInfo.java

```diff
@@ -115,11 +115,11 @@ private static int computeCreditsRequired(List<Requirement> requirementList) {
             return Optional.empty();
```

Comment:
good  job Teri, thanks for fixing

Reply:
sorry i had to touch this method abit as well, cos last week i was fixing some cap stuff and some bugs here too, but i'll add @author for you

Reply:
no probs, thanks!